mlton/backend/restore.sig

```diff
@@ -0,0 +1,22 @@
+(* Copyright (C) 1999-2005 Henry Cejtin, Matthew Fluet, Suresh
+ *    Jagannathan, and Stephen Weeks.
+ * Copyright (C) 1997-2000 NEC Research Institute.
+ *
+ * MLton is released under a HPND-style license.
+ * See the file MLton-LICENSE for details.
+ *)
+
+signature RESTORE_R_STRUCTS =
+   sig
+      include RSSA
+   end
+
+signature RESTORE_R =
```

Comment:
I think that this should be named `RESTORE_RSSA`.  I know that we have `RESTORE` (for SSA) and `RESTORE2` (for SSA2), but this seems unnecessarily opaque.